src/platform/meteorlake/include/platform/lib/ams.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ */
+
+#ifdef __SOF_LIB_AMS_H__
+
+#ifndef __PLATFORM_LIB_AMS_H__
+#define __PLATFORM_LIB_AMS_H__
+
+/* max number of message UUIDs */
+#define AMS_SERVICE_UUID_TABLE_SIZE 16
+/* max number of async message routes */
+#define AMS_ROUTING_TABLE_SIZE 16
+/* Space allocated for async message content*/
+#define AMS_MAX_MSG_SIZE 0x1000
```

Comment:
should this be a sizeof(message size) * MAX NUM MESSAGES or something?

can this be less than 4k or is there any physical limitation here (SRAM power controls, etc).

Reply:
Since there are no usecases yet, I'm not sure about this value. I'm pretty sure that this has to do with page size and optimization, but the exact real scenario payloads are not known at the moment. This is a safe value copied from closed source intel solution, so it's guaranteed to fit all the other components that are based on this implementation. I know it's not the best argument for an open source solution, but that's the best one I have.

Reply:
Then if it's a value that was experimented with before, add a comment to that effect. Otherwise this seems completely arbitrary and as you write clearly not the best way to make progress with open-source reviews